### ISSUE TITLE
Does not recognize GPU

### ISSUE THREAD

**Issue by erikwakefield** [Djdefrag/RealScaler#24]
I have an EVGA 2080 Super with the latest drivers (ver. 565.90 as of this post) but the system monitor shows no GPU activity during upscaling regardless of which GPU I select from the dropdown.

**Comment by erikwakefield** [comment]
Yes, I'm looking at task manager, it doesn't seem to make a difference when i select other GPU's from the dropdown. In the main task list i see Python using 0% of the GPU, i don't have as many options as you do for charts and i'm only seeing a temp increase of a degree or two, which seems to be tied to snipping tool. 
![Screenshot 2024-10-08 095405](https://github.com/user-attachments/assets/55b8b32d-77e7-46d3-a345-1c89566c9f9e)

**Comment by Djdefrag** [comment]
HI my friend,

yes, is using only your cpu, try install following libraries

1.  install Visual-C-Runtimes: 
   - https://www.techpowerup.com/download/visual-c-redistributable-runtime-package-all-in-one/

2.  install Directx:
   - https://www.microsoft.com/it-it/download/details.aspx?id=35

**Comment by erikwakefield** [comment]
I installed the Visual-C runtimes and I have DirectX 12, still no change. I'm also running Windows 11 FWIW

**Comment by Djdefrag** [comment]
This is really strange my friend,

are you using Github code?

**Comment by erikwakefield** [comment]
Yes, everything came from Github

**Comment by Djdefrag** [comment]
Hi my friend, 

have you tried enable Hardware Accelerated GPU scheduling?

You can find this option in Windows > Settings > Graphic Settings menu
